srcpkgs/tap-plugins/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'tap-plugins'
+pkgname=tap-plugins
+version=1.0.0
+revision=1
+build_style=gnu-makefile
+makedepends="ladspa-sdk"
+short_desc="Tom's LADSPA Plugins"
+maintainer="Olga Ustuzhanina <me@laserbat.pw>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/tomszilagyi/tap-plugins/"
+distfiles="https://github.com/tomszilagyi/${pkgname}/archive/v${version}.tar.gz"
+checksum=ea94d8b20d8056e1b9ec6d252846713780fd2a268f3a3a772c362a10950d80cd
+
+do_configure() {
+	sed 's|/usr/local/|$(DESTDIR)/usr/|' -i Makefile
+}
+
+do_build() {
+	make ${makejobs}
```

Comment:
Can you try setting `make_use_env=yes` globally and remove the custom `do_build`

Reply:
Done, it works! Thank you :)